### ISSUE TITLE
beauty tag is missing for shop=beauty Ulta Beauty (USA)

### ISSUE THREAD

**Issue by matkoniecz** [osmlab/name-suggestion-index#2663]
Is there one that apply to every single location for this poi?
What would be appropriate value for it?

This brand may be described at https://en.wikipedia.org/wiki/Ulta_Beauty

entry data:
```
{
  "countryCodes": [
    "us"
  ],
  "matchNames": [
    "ulta"
  ],
  "tags": {
    "brand": "Ulta Beauty",
    "brand:wikidata": "Q7880076",
    "brand:wikipedia": "en:Ulta Beauty",
    "name": "Ulta Beauty",
    "shop": "beauty"
  }
}
```

@1ec5, @Adamant36, @maxerickson

**Comment by tas50** [comment]
Ulta doesn't really make sense as a beauty shop at all. Beauty shops offer a service like hair & nails. Ulta is a retail store that mostly sells cosmetics. It should be retagged as a cosmetics shop.

**Comment by tas50** [comment]
To quote my wife "it's Sephora, but not as nice"

**Comment by sinclarius** [comment]
@tas50 Ulta does offer beauty services, though, in addition to selling cosmetics. https://www.ulta.com/beautyservices/

**Comment by tas50** [comment]
@sinclarius TIL. Does this seem about right? https://github.com/osmlab/name-suggestion-index/pull/2675

**Comment by sinclarius** [comment]
@tas50  Sure does! Just a question for my own clarification: how are these sub-tags of "beauty" ("cosmetics", "eyebrows", etc) represented in iD? Will Ulta appear as an option whenever the word "supplies" is entered in the search field? 

![image](https://user-images.githubusercontent.com/3949486/58418323-c1427400-8044-11e9-910a-d3fac33c62f7.png) (via  #2675)

**Comment by tas50** [comment]
I'm not really sure how this all gets exposed in search. The default search in openstreetmap isn't the best. I think the idea here is to add the data that could be exposed by better search sometime in the future.

**Comment by matkoniecz** [comment]
Fixed by #2675
